vsce/server/src/server.ts

```diff
@@ -286,7 +288,34 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
 	const reachPath = (exeLoc == '' || exeLoc == './reach')
 		? path.join(workspaceFolder, "reach")
 		: exeLoc;
+
+	if (theCompilerIsCompiling) {
+		console.debug(
```

Comment:
My comment in the call was that you could do `doAnotherCompile = true` here

Reply:
I see. 1304276 should address this.

Thank you for adding the suggestions to a specific line: that makes things a lot easier for me.